[feature] aws-single-page-static-site: allow disabling the custom error responses

palasha opened the pull request:
This is useful if we dont want this behavior.

Test Plan: lets see if it works.

alldoami commented on aws-single-page-static-site/variables.tf:
```diff
@@ -68,3 +68,10 @@ variable "path_pattern" {
+variable "custom_error_response_codes" {
+  type = list(number)
+  description = "The http response codes for which to return the default index page."
+  default = [404, 403, 502]
```
```suggestion
  default = [404, 403, 503]
```
from what I see I think it's supposed to be `503`?

alldoami commented:
Also looks like you need to do a `make fmt`

palasha commented:
- Fixed incorrect response code
- make fmt
- verified this is working as expected on www.comealong.org

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/cztack/compare/40e6d91a05df9206ea596490c3ea341c7ccddbec..7431c50f27c0a6c480965b101a0daa8a5f419e2c).

ryanking commented:
@palasha can we merge this?
